docs/source/index.rst

```diff
@@ -6,12 +6,88 @@
+As a result, exploring a system by unbiased MD is insufficient to obtain good statistics, especially if the free-energy landscape is separated by high barriers.
+To address this well know drawback enhanced-sampling methods have been established.
+Typically, a configuration can be reduced to a collective variable (order parameter) and the simulation is biased based on these collective variables.
+The challenge for computer simulations is that i) almost every interesting system has its own collective variable description and ii) the implementation
+of collective variables and methods has run efficiently on modern computers, to allow reasonable insights into the observable of interest.
+
+No compromises in usability and speed for enhanced-sampling methods!
 --------------------------------------------------------------------
 
-PySAGES provides an extensible framework to compute collective variables and to perform
-enhanced-sampling molecular dynamics simulations to discover reaction pathways and estimate
-free energies.
+PySAGES addresses these challenges by offering a python interface between highly optimized simulation engines and the researcher to implement collective variables and enhance-sampling methods.
+However, PySAGES already provides an extensible framework to compute collective variables and to perform enhanced-sampling molecular dynamics simulations to discover reaction pathways and estimate free energies.
+Most research objectives are achievable by using these implemented collective variables and methods.
+PySAGES connects these methods automatically to a highly optimized simulation back.
+We support two backends at the moment: `HOOMD-blue <https://glotzerlab.engin.umich.edu/hoomd-blue>`__ and
+`OpenMM <http://openmm.org/>`__.
+Both engines have in common that they offer a python interface to the user and implement the simulation on GPUs for best performance.
+PySAGES interacts with both backends directly on the GPU memory, no copying between GPU and host memory is required.
+This approach allows biased simulations without slowing the backend simulation engines down.
+Still, PySAGES implements all methods and collective variables as pure python for access and modification.
+
+
+Getting Started
+---------------
+
+PySAGES is designed to seamlessly integrate with your existing simulations.
+For an easily accessible start we prepared examples in Google Colab documents, they enable easy access to a working simulation environment including GPU hardware.
+For an example of using PySAGES with HOOMD-blue checkout `this <https://colab.research.google.com/github/SSAGESLabs/PySAGES/blob/main/examples/hoomd-blue/Umbrella_Integration.ipynb>`__ example for simple umbrella integration.
+If you are working with OpenMM we offer `this <https://colab.research.google.com/github/SSAGESLabs/PySAGES/blob/main/examples/openmm/Harmonic_Bias.ipynb>`__ notebook, which explores the Alanine Dipeptide dihedral angle via harmonic biasing.
```

Comment:
if I'm not mistaken, here also, as it is usually done, the 2 dihedral angles of adp are biased. but I could not open the colab at the moment to double-check. 

If I'm right, this may be reformulated to:
"which explores the free energy landscape of the conformational space of alanine dipeptide via harmonic biasing."

Reply:
you are right. I changed the wording.